src/patterns/FavoritesCollectionTile/FavoritesCollectionTile.tsx

```diff
@@ -196,68 +219,105 @@ const FavoritesCollectionTile = memo(
                 )}
               </>
             </div>
-            <Component href={href} className={classnames(`${baseClassName}__media-link`, linkClassName)} tabIndex={0}>
-              {isCountEmpty && variant === 'favorites' && (
-                <div className={`${baseClassName}__media-container`} data-testid="favorites" aria-label="Favorites">
-                  <div className={classnames(`${baseClassName}__empty`, `${baseClassName}__empty--bg`)}>
-                    <div className={`${baseClassName}__empty__content`}>
-                      <Icon
-                        icon="Favorite"
-                        width={iconSize}
-                        height={iconSize}
-                        color="$dark-gray"
-                        className={`${baseClassName}__icon`}
-                      />
-                      <div className={`${baseClassName}__text`}>{emptyFavoritesText}</div>
+            {isCreateVariant ? (
```

Comment:
these chained if blocks are a bit difficult to parse, I wonder if extracting each section to a smaller reusable component would help `<CreateFirstListButton`